icevision/models/mmseg/lightning/model_adapter.py

```diff
@@ -0,0 +1,67 @@
+__all__ = ["MMDetModelAdapter"]
+
+from icevision.imports import *
+from icevision.utils import *
+from icevision.metrics import *
+from icevision.engines.lightning.lightning_model_adapter import LightningModelAdapter
+
+from icevision.models.mmdet.common.bbox import convert_raw_predictions
+
+
+class MMDetModelAdapter(LightningModelAdapter, ABC):
+    """Lightning module specialized for EfficientDet, with metrics support.
+
+    The methods `forward`, `training_step`, `validation_step`, `validation_epoch_end`
+    are already overriden.
+
+    # Arguments
+        model: The pytorch model to use.
+        metrics: `Sequence` of metrics to use.
+
+    # Returns
+        A `LightningModule`.
+    """
+
+    def __init__(self, model: nn.Module, metrics: List[Metric] = None):
+        super().__init__(metrics=metrics)
+        self.model = model
+
+    @abstractmethod
+    def convert_raw_predictions(self, batch, raw_preds, records):
```

Comment:
pure python question. How can this work without a `return` or `pass` statement inside the function?

Reply:
What function gets invoked [here](https://github.com/airctic/icevision/pull/934/files#diff-b52b136bda8d73b70540f7af32ce576d54b6644eb6c25ba4f08ee46df367a760R55) exactly?